Sources/CreditCardScanner/CreditCardScannerViewController.swift

```diff
@@ -71,8 +71,9 @@ open class CreditCardScannerViewController: UIViewController {
         fatalError("Not implemented")
     }
 
-    override open func viewDidLoad() {
-        super.viewDidLoad()
+    override open func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.analyzer = ImageAnalyzer(delegate: self)
```

Comment:
Sorry for the delay, I must have missed this! This implementation results in layoutSubviews and such being called again, which is not desirable. Is re-initializing ImageAnalyzer the best way to address navigation controllers?

Reply:
Omg. I made a mistake.
I needed the feature that the subview which has the card scanner should be called again, which consists of navigation views. The re-initializing code line is from my habbit. I tested it without the line.

Reply:
But anyway, thanks for this library. I'm using this very usefully.

Reply:
Also if you feel like this is not desirable, you can just close this pr. Thanks.